lib/config-generator.js

```diff
@@ -141,34 +141,34 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
                 "browser,node": "globals: {...globals.browser, ...globals.node}"
             };
 
-            exportContent += `  {languageOptions: { ${envContent[this.answers.env.join(",")]} }},\n`;
+            exportContent += `  { files: ["**/*.js"], languageOptions: { ${envContent[this.answers.env.join(",")]} } },\n`;
```

Comment:
Restricting globals to `.js` files might not be desirable, particularly when additional extensions are included in linting based on the answers. For example, browser globals wouldn't be available in `.jsx` files.

Reply:
Good point. I'll make that update.